src/app/containers/Figure/index.jsx

```diff
@@ -25,8 +25,8 @@ const renderImage = (imageToRender, lazyLoad) =>
 const renderCopyright = copyright =>
   copyright ? <Copyright>{copyright}</Copyright> : null;
 
-const renderCaption = (block, type) =>
-  block ? <Caption block={block} type={type} /> : null;
+const renderCaption = block =>
+  block ? <Caption block={block} type="image" /> : null;
```

Comment:
What the reason this has been hardcoded to `"image"`? 

The type was previously passed to Caption container, then to `chooseOffscreenText` which switched between which type of offscreen text to use for each service.

Reply:
@sareh suggested that because it's a choice between "video caption" and "image caption" and 'caption' that it makes sense to do there, but I may have misunderstood her suggestion.

Are there other types of caption potentially?

Reply:
The switching logic makes sense in Caption container, that isn't what I'm questioning. What I'm pointing out is that Figure container which calls Caption container is now hardcoded to the string "image" meaning that any use of Figure container is now explicitly getting the image offscreen text, rather than the `type` it gets passed

Reply:
@pjlee11 - @oluoluoxenfree has updated it now to move the declaration of type to the Image/Video containers.